pkg/api/groups.go

```diff
@@ -489,7 +489,11 @@ func (api *API) GetGroupVersionCountTimeline(groupID string, duration string) (m
 	WITH time_series AS (SELECT * FROM generate_series(now() - interval '%[1]s', now(), INTERVAL '%[2]s') AS ts),
 		 recent_instances AS (SELECT instance_id, (CASE WHEN last_update_granted_ts IS NOT NULL THEN last_update_granted_ts ELSE created_ts END), version, 4 status FROM instance_application WHERE group_id=$1 AND last_check_for_updates >= now() - interval '%[1]s' AND %[3]s ORDER BY last_update_granted_ts DESC),
 		 instance_versions AS (SELECT instance_id, created_ts, version, status FROM instance_status_history WHERE instance_id IN (SELECT instance_id FROM recent_instances) AND status = 4 UNION (SELECT * FROM recent_instances) ORDER BY created_ts DESC)
-	SELECT ts, (CASE WHEN version IS NULL THEN '' ELSE version END), sum(CASE WHEN version IS NOT null THEN 1 ELSE 0 END) total FROM (SELECT * FROM time_series LEFT JOIN LATERAL(SELECT distinct ON (instance_id) instance_Id, version, created_ts FROM instance_versions WHERE %[4]s AND created_ts <= time_series.ts ORDER BY instance_Id, created_ts DESC) _ ON true) AS _
+	SELECT ts, (CASE WHEN version IS NULL THEN '' ELSE version END), sum(CASE WHEN version IS NOT null THEN 1 ELSE 0 END) total FROM (
+	SELECT * FROM time_series LEFT JOIN (SELECT distinct ON (instance_id) instance_Id,
+	version, created_ts FROM instance_versions WHERE %[4]s) _
+	ON created_ts <= time_series.ts ORDER BY instance_Id, created_ts DESC
```

Comment:
Thanks for reformatting this query a bit. Could you indent this block further?

```suggestion
		SELECT * FROM time_series LEFT JOIN (SELECT distinct ON (instance_id) instance_Id,
		version, created_ts FROM instance_versions WHERE %[4]s) _
		ON created_ts <= time_series.ts ORDER BY instance_Id, created_ts DESC
```